manager/controlapi/cluster.go

```diff
@@ -58,6 +58,10 @@ func validateClusterSpec(spec *api.ClusterSpec) error {
 		}
 	}
 
+	if spec.Annotations.Name != store.DefaultClusterName {
+		return grpc.Errorf(codes.PermissionDenied, "Modification of cluster name is node allowed")
```

Comment:
s/node/not/ 😉 

Also can you lowercase `Modification`?

Reply:
Also wondering;

- is this validation done on _create_ / _init_ as well?
- should "empty" values be updated to `store.DefaultClusterName`? (as is done in https://github.com/moby/moby/pull/28914)

Reply:
> is this validation done on create / init as well?

Creation name is hardcoded, so I don't think a validation is needed there.

>should "empty" values be updated to store.DefaultClusterName

Not sure what the motivation behind this is, but its probably better to be explicit and fail if it changes. I believe clients are expected to perform a read-modify-write ?

I guess setting the default would have been much cleaner if there was a way to specify the default value during init (e.g. inline init or in the constructor).